### PR TITLE
Add print-js w/ npm auto-aupdate

### LINKED ISSUE #10234: [Request] Add print.js
**Library name:** print.js
**Git repository url:** https://github.com/crabbly/Print.js
**npm package url(optional):** print.js
**License(s):** MIT
**Official homepage:** http://printjs.crabbly.com/

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/40787085-request-add-print-js?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by sashberd**
close #10234, cc @crabbly

Pull request for issue: #10234
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/crabbly/Print.js
 * Official website (optional, not the repository): http://printjs.crabbly.com/
 * NPM package url (optional): https://www.npmjs.com/package/print-js
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required): 
   - Count of watchers: 18
   - Count of stars: 292
   - Count of forks: 68
 * NPM download stats (optional):
   - Downloads in the last day: 29
   - Downloads in the last week: 478
   - Downloads in the last month: 1505

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sashberd** [comment]
@xtuc @MattIPv4 FYR

File: ajax/libs/print-js/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    },
+    {
+      "basePath": "src",
+      "files": [
+        "css/print*"
```
Comment (MattIPv4) [inline ajax/libs/print-js/package.json]: No files appear to match this in the package:

```
node_modules/print-js
├── LICENSE
├── README.md
├── dist
│   ├── print.css
│   ├── print.js
│   └── print.map
├── package.json
└── src
    ├── index.d.ts
    ├── index.js
    ├── js
    │   ├── browser.js
    │   ├── functions.js
    │   ├── html.js
    │   ├── image.js
    │   ├── init.js
    │   ├── json.js
    │   ├── modal.js
    │   ├── pdf.js
    │   ├── print.js
    │   └── raw-html.js
    └── sass
        ├── index.scss
        ├── modules
        │   └── _colors.scss
        └── partials
            ├── _modal.scss
            └── _spinner.scss
```

File: ajax/libs/print-js/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "print-js",
+  "filename": "print.min.js",
```
Comment (MattIPv4) [inline ajax/libs/print-js/package.json]:
```suggestion
  "filename": "print.js",
```
Only `print.js` is included in the package, not `print.min.js`.

File: ajax/libs/print-js/package.json (hunk 1 above)
Comment (sashberd) [reply ajax/libs/print-js/package.json]: For versions below 1.0.46 this one will get css file inside css folder

File: ajax/libs/print-js/package.json (hunk 2 above)
Comment (sashberd) [reply ajax/libs/print-js/package.json]: updated

**Pull request merged**